frame/wormhole/issuing/s2s/src/lib.rs

```diff
@@ -170,18 +170,20 @@ pub mod pallet {
 						&name,
 						&symbol,
 						option.decimal,
-						backing,
+						backing_address,
 						token_info.address,
```

Comment:
Could we generate new symbol and name here? We will require less parameters. and we do not have store prefix and pass backing_chain_name in contract template.

Reply:
Yes we can do this.